<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>390</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gray_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>430</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Processing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eqhist_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>EqHist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>480</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close App</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blur_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="median_blur_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussian_blur_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bilateral_blur_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Bilateral Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threshold_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threshold_otsu_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold(otsu)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dilate_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="erode_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>420</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Erode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morph_open_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morph_closing_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Closing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morph_gradient_button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gradient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morph_tophat_button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>420</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>TopHat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morph_blackhat_button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>450</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>BlackHat</string>
    </property>
   </widget>
   <widget class="QLabel" name="uiImg1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="uiImg2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="uiImg3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Denoise Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="uiImg4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="uiImg5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Morphology Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>520</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ExecuteTime : </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
